src/app/map/components/time-slider/time-slider.component.ts

```diff
@@ -144,9 +144,9 @@ export class TimeSliderComponent implements OnInit, OnChanges {
     const position = this.findPositionOfDate(date);
     if (position !== undefined) {
       if (changedMinimumDate) {
-        this.effectiveMinimumDateIndex = position;
+        this.firstSliderPosition = position;
       } else {
-        this.effectiveMaximumDateIndex = position;
+        this.secondSliderPosition = position;
```

Comment:
As far as I understood, changing the date from the datepicker should not change the actually selected date, but the range in which dates can be selected. 

```suggestion
        this.maximumDateIndex= position;
```

Same for the line above, use `this.minimumDateIndex` instead. 
The selecteed date will adjust if needed

Reply:
Current state is preferred. Must be specifically evaluated again during future timeslider revision.